src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -69,11 +70,8 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
     this.options = ComponentOptions.initComponentOptions(element, QuerySuggestPreview, options);
 
     if (!this.options.resultTemplate) {
```

Comment:
BTW do we also apply the default template if the specified `resultTemplate` cannot be found?

Reply:
Not so far, I didn't know about this case. Should I apply the default template if `resultTemplate` cannot be found, add an error message or leave it as-is?

Reply:
I personally think we should output an error message; but maybe this excpetion is already handled elsewhere?

Reply:
It currently throws an error message, but only on `instantiateElement`. I'll add one then.